modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-api/bnd.bnd

```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay Portal Workflow Metrics REST API
 Bundle-SymbolicName: com.liferay.portal.workflow.metrics.rest.api
-Bundle-Version: 8.0.2
+Bundle-Version: 8.1.0
```

Comment:
@MarinhoFeliphe We have a specific commit pattern for this type of changes("LPS-xxxx Baseline OR LPS-xxxx Semver").